### ISSUE TITLE
consider dynamic properties on nodes

### ISSUE THREAD

**Issue by jtauber** [jtauber/sgf#6]
It might be nice if instead of things like `node.properties["SZ"]`, `node.SZ` would just work.

It could potentially return `None` if the property isn't on the node, so you could do stuff like

```
size = game.root.SZ or 19
```

(assuming #3 #4 and #5 are also done) instead of

```
size = int(game[0].properties.get("SZ", ["19"])[0])
```

To be precise, #3 would allow `game.root` for `game[0]`, #4 would allow `properties.get("SZ", "19")` instead of `properties.get("SZ", ["19"])[0]`, #5 would allow the `int(...)` to be dropped and the `"19"` default be written as `19`, and this (#6) would support `.SZ` instead of `.properties.get("SZ")`

**Comment by wrongu** [comment]
I think it would be just as convenient to implement `__getitem__` on nodes that gets their properties, or even have your nodes extend `dict` objects and get rid of `properties` altogether

**Comment by jtauber** [comment]
Yeah, let me do one of those. #3 is now merged but I still think #4 and #5 would be nice as well.
